src/components/header/mobile/MobileNav.vue

```diff
@@ -51,6 +51,7 @@
               <span class="text--link">{{ $t('sidenavi.forum') }}</span>
             </div>
           </a>
+          <a href="" target="_blank">{{ $t('sidenavi.governance') }}</a>
```

Comment:
Is it ok to display governance but no URL? @bobo-k2 @gtg7784 
![image](https://github.com/user-attachments/assets/e148a1e2-6b2f-4a10-88cd-361f09e627ac)

Reply:
This has been resolved in [this](https://github.com/AstarNetwork/astar-apps/pull/1409#pullrequestreview-2398059367) PR.